tests/core/test_evaluation_parameters.py

```diff
@@ -133,6 +133,8 @@ def test_parser_timing():
 
 def test_math_evaluation_paramaters():
     assert parse_evaluation_parameter("sin(2*PI)") == math.sin(math.pi * 2)
+    assert parse_evaluation_parameter("cos(2*PI)") == math.cos(math.pi * 2)
+    assert parse_evaluation_parameter("tan(2*PI)") == math.tan(math.pi * 2)
```

Comment:
Just wanted more coverage here. Not part of the ticket but sneaking it in while I'm in this file.